pyproject.toml

```diff
@@ -77,6 +82,44 @@ test = "./tools/GeneratePythonLibrary.sh --test local"
 recursive = true
 
 [tool.black]
-target-version = ["py39"]
+target-version = ["py311"]
 line-length = 120
 skip-string-normalization = true
+
+[tool.mypy]
+strict = false
+show_error_codes = true
+follow_imports = 'silent'
+exclude = '''(?x)(
+    /src/
+)'''
+python_version = 3.12
+
+# Start off with these
```

Comment:
Is the goal to eventually use the default `mypy` config like we do for `kagglehub`?

https://github.com/Kaggle/kagglehub/blob/b7317a2448f68e766e04a05718ad55823fc35a65/pyproject.toml#L91